cmd/autoowners/main.go

```diff
@@ -87,10 +87,13 @@ func loadRepos(configRootDir string, blocked blocklist, configSubDirs []string,
 			if org == githubOrg && repo == githubRepo {
 				continue
 			}
-			repoConfigDirs := make([]string, 0, len(configSubDirectories))
+			var repoConfigDirs []string
 			for _, sourceSubDir := range configSubDirectories {
 				repoConfigDirs = append(repoConfigDirs, filepath.Join(sourceSubDir, org, repo))
 			}
+			for _, extraDir := range extraDirs {
+				repoConfigDirs = append(repoConfigDirs, filepath.Join(extraDir, org, repo))
```

Comment:
What if it doesn't exist?

Reply:
It gets filtered out below, as with all the others. The test handles this - only a dir for one of the two repos exists in the extra directory tree.